pkg/storage/sqlstorage/migrations_test.go

```diff
@@ -420,7 +418,7 @@ var postMigrate = map[string]func(t *testing.T, store *sqlstorage.Store){
 	},
 }
 
-func TestMigrates(t *testing.T) {
+func TestAllMigrations(t *testing.T) {
 
 	if ledgertesting.StorageDriverName() != "sqlite" {
```

Comment:
With those changes, it is still normal to skip this test if not SQLite ?

Reply:
@antoinegelloz Good question, now, it should works with PG, i will test.

Reply:
And yes, it works.
Updated!